utils/utils.go

```diff
@@ -0,0 +1,30 @@
+package utils
```

Comment:
Need to add a copyright on all published files

Reply:
added copyright